doc/examples/features_detection/plot_remove_objects.py

```diff
@@ -0,0 +1,44 @@
+"""
+================
+Removing objects
+================
+
+scikit-image supports several ways to remove objects inside N-dimensional
+images. In this context, "objects" (and "holes") are defined as groups of
+connected samples that are distinct from the background. A binary image can
```

Comment:
Maybe say what "connected means".

Presumably, not only binary images can contain several disconnected objects.